FlorinPosea/teme/3.5.react-rgb-color-game/src/App.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import ColorSquare from "./components/color-square/ColorSquare";
+import "./App.css";
+
+const DEFAULT_SQUARES = 6;
+var winningcolor = "";
+var msg = "";
+
+function App() {
+  const [newList, setNewList] = useState(true);
+  const [colorsList, setColorsList] = useState([]);
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedColor, setSelectedColor] = useState();
```

Comment:
Nu e folosit bine setSelectedColor. Din nou e setat state-ul, dar nu e folosit. setSelectedColor(getWinningColor()) ar fi probabil o idee buna. Pentru ca tu ai nevoie sa stii in state, care e culoarea castigatoare. 
![image](https://user-images.githubusercontent.com/45266485/201000875-862e7c08-1fb0-4e02-bfc4-79ffd830dea9.png)

Reply:
Aici ca sa clarific, setSelectedColor, ar putea devenii setWinningColor, nu trebuie neaparat sa tinem minte care culoare este selectata, ci atunci cand dam click pe o culoare sa verificam daca e egala cu culoarea castigatoare, si daca este atunci afisam "msg".